docs/source/ddp_comm_hooks.rst

```diff
@@ -22,14 +22,16 @@ How to Use a Communication Hook?
 To use a communication hook, the user just needs to let the DDP model register
 the hook before the training loop as below.
 
-:func:`torch.nn.parallel.DistributedDataParallel.register_comm_hook`.
+:func:`torch.nn.parallel.DistributedDataParallel.register_comm_hook`
     :noindex:
 
 Default Communication Hooks
 ---------------------------
 
 Default communication hooks are simple **stateless** hooks, so the input state
 in ``register_comm_hook`` is either a process group or ``None``.
+The input ``bucket`` is a :class:`torch.distributed.GradBucket` object,
```

Comment:
Is the `GradBucket` class documented somewhere?

Reply:
> Is the `GradBucket` class documented somewhere?

Similar to `register_comm_hook`, won't the definition be found in https://pytorch.org/docs/stable/generated/torch.nn.parallel.DistributedDataParallel.html?

Or do we need to provide a separate `autoclass:: torch.distributed.GradBucket`?

Reply:
I think we may need the autoclass. Since `GradBucket` is now public, could you post what the docs look like and ensure `GradBucket` and its methods are appropriately rendered?

Reply:
Added to the description of this PR.